packages/polyfills/types/global.d.ts

```diff
@@ -11,3 +11,5 @@ type LiteralUnion<
     : U extends number ? number
     : never,
 > = U | (T & Nothing)
+
+type HexString = `0x${string}`
```

Comment:
I'll not suggest you to add it in the polyfills package, but you can go if you really want.